CliClient/locales/es_ES.po

```diff
@@ -463,15 +467,14 @@ msgstr "Descargando recursos..."
 msgid "Cancelling... Please wait."
 msgstr "Cancelando... Por favor espere."
 
-#, fuzzy
 msgid ""
 "<tag-command> can be \"add\", \"remove\" or \"list\" to assign or remove "
 "[tag] from [note], or to list the notes associated with [tag]. The command "
 "`tag list` can be used to list all the tags (use -l for long option)."
 msgstr ""
-"<tag-command> puede ser \"add\", \"remove\" o \"list\" para asignar o "
-"eliminar [tag] de [note], o para listar las notas asociadas con [tag]. El "
-"comando `tag list` puede ser usado para listar todas las etiquetas."
+"<tag-command> can be \"add\", \"remove\" or \"list\" to assign or remove "
```

Comment:
Why did you change the Spanish text to English?

Reply:
Oops! That was a mistake.

Reply:
Thanks for the update.

LGTM.